python/ray/dashboard/modules/job/job_supervisor.py

```diff
@@ -173,6 +173,7 @@ def _exec_entrypoint(self, env: dict, logs_path: str) -> subprocess.Popen:
         # Open in append mode to avoid overwriting runtime_env setup logs for the
         # supervisor actor, which are also written to the same file.
         with open(logs_path, "a") as logs_file:
+            self._logger.info(f"Running entrypoint for job {self._job_id}: {self._entrypoint}\n")
```

Comment:
I haven't touched this code in a long while, but I believe `self._logger` goes to a log file for this supervisor actor, not the log file for the job itself. You'll want to write this to the `logs_file` that was opened a line above instead.